engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/preparation/JobPreparationActor.scala

```diff
@@ -153,6 +157,26 @@ class JobPreparationActor(workflowDescriptor: EngineWorkflowDescriptor,
     }
   }
 
+  private def updateRuntimeMemory(runtimeAttributes: Map[LocallyQualifiedName, WomValue]): Map[LocallyQualifiedName, WomValue] = {
+    def doubleRuntimeMemory(multiplier: Int): Map[LocallyQualifiedName, WomValue] = {
+      runtimeAttributes.get(RuntimeAttributesKeys.MemoryKey) match {
+        case Some(WomString(memory)) =>
+          MemorySize.parse(memory) match {
+            case Success(mem) =>
+              val memString = MemorySize(mem.amount * multiplier, mem.unit).toString
```

Comment:
ToL: Ah. I see. There's a bit of `String` -> `Double` (* `Int`) -> `String` conversion, and `mem.amount` is already a `Double`.